src/pages/Login.tsx

```diff
@@ -242,21 +242,32 @@ const Login: React.FC = () => {
         setSentOtpLoading(false);
         alert("Phone Number signin Failed " + authRes);
       }
-    } catch (error) {
-      console.log("Phone Number signin Failed ");
-      setSpinnerLoading(false);
-      setSentOtpLoading(false);
-      alert("Phone Number signin Failed " + error);
-      console.log(
-        "window.recaptchaVerifier",
-        // window.recaptchaVerifier,
-        recaptchaVerifier!
-      );
-
-      // //@ts-ignore
-      recaptchaVerifier!.clear();
-      // //@ts-ignore
-      // window.recaptchaVerifier.clear();
+    }catch (error) {
+        console.log("Phone Number signin Failed ");
+        setSpinnerLoading(false);
+        setSentOtpLoading(false);
+    
+        if (typeof error === 'string') {
```

Comment:
if possible get error code from error type and show the error message according to error code